cli/utils/stage-library-ts.js

```diff
@@ -66,10 +66,10 @@ function getFileContents(filePath) {
   switch (path.extname(filePath)) {
     case '.scss':
       contents = compileSass(filePath);
-      break;
+    break;
```

Comment:
It's unclear how this got in to start with, but the linter complained to be about the spacing.